README.md

```diff
@@ -45,6 +45,14 @@ Use them to instead of `trace()`, `Sys.exit()`, etc, for maximum compatibility a
 - `travix.Logger.println(string)`: Print a string with newline
 - `travix.Logger.exit(exitCode)`: Exit the process
 
+If you want to be able to run your tests e.g. locally without Travix dependency use conditional compilation like this:
+```haxe
+#if travix
```

Comment:
FWIW `#if travix import travix.Logger.println in trace; #end` keeps the conditional compilation more limited. 

That said, I do wonder why you wouldn't want to use travix locally. Having two different ways to run the tests adds complexity.